static/templates/set_status_overlay.hbs

```diff
@@ -1,30 +1,26 @@
-<div class="user_status_overlay overlay new-style" data-overlay="user_status_overlay" aria-hidden="true">
-    <div class="overlay-content modal-bg">
-        <div class="user-status-header">
-            <h1>{{t "Set a status" }}</h1>
-            <div class="exit">
-                <span class="exit-sign">&times;</span>
-            </div>
-        </div>
-        <div class="modal-body">
-            <label for="user_status">{{t "Status message" }}</label>
-            <input type="text" class="user_status" maxlength="60" />
-            <button type="button" class="btn clear_search_button" id="clear_status_message_button" disabled="disabled">
-                <i class="fa fa-remove" aria-hidden="true"></i>
-            </button>
-        </div>
+<div id="set_user_status_modal" class="modal modal-bg hide fade new-style" tabindex="-1" role="dialog" aria-labelledby="set_user_status_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close close-modal-btn" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+        <h3 id="set_user_status_modal_label">{{t "Set status" }} </h3>
+    </div>
+    <div class="modal-body">
+        <label for="user_status">{{t "Status message" }}</label>
+        <input type="text" class="user_status" maxlength="60" />
+        <button type="button" class="btn clear_search_button" id="clear_status_message_button" disabled="disabled">
+            <i class="fa fa-remove" aria-hidden="true"></i>
+        </button>
         <div class="user-status-options">
             <button type="button" class="button no-style user-status-value">{{t "In a meeting" }}</button>
             <button type="button" class="button no-style user-status-value">{{t "Commuting" }}</button>
             <button type="button" class="button no-style user-status-value">{{t "Out sick" }}</button>
             <button type="button" class="button no-style user-status-value">{{t "Vacationing" }}</button>
             <button type="button" class="button no-style user-status-value">{{t "Working remotely" }}</button>
         </div>
```

Comment:
`user-status-options` was very weirdly outside `modal-body`; I fixed that while merging this, since that's clearly the design intent (and results in preserving the overall visual design of the widget, without a large space between the text box and these options).